napari/_qt/layer_controls/qt_image_controls_base.py

```diff
@@ -259,3 +262,33 @@ def reset():
             range_btn.setFixedWidth(65)
             range_btn.clicked.connect(layer.reset_contrast_limits_range)
             self._layout.addWidget(range_btn, alignment=Qt.AlignBottom)
+
+
+def range_to_decimals(range_, dtype):
```

Comment:
this is fine, but I assume you meant to delete this code from QContrastLimitsPopup above?  currently it's duplicated

Reply:
lol yes - and replace it there too. will fix - thanks!!

Reply:
done